frontend/packages/operator-lifecycle-manager/src/components/catalog-source.tsx

```diff
@@ -348,33 +364,45 @@ const CatalogSourceList: React.FC<TableProps> = (props) => {
         transforms: [sortable],
         props: { className: tableColumnClasses[0] },
       },
+      {
+        title: t('catalog-source~Status'),
+        sortField: 'status',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[1] },
+      },
       {
         title: t('catalog-source~Publisher'),
         sortField: 'publisher',
         transforms: [sortable],
-        props: { className: tableColumnClasses[1] },
+        props: { className: tableColumnClasses[2] },
       },
       {
         title: t('catalog-source~Availability'),
         sortField: 'availabilitySort',
         transforms: [sortable],
-        props: { className: tableColumnClasses[2] },
+        props: { className: tableColumnClasses[3] },
       },
       {
         title: t('catalog-source~Endpoint'),
         sortField: 'endpoint',
         transforms: [sortable],
-        props: { className: tableColumnClasses[3] },
+        props: { className: tableColumnClasses[4] },
+      },
+      {
+        title: t('catalog-source~Registry Poll Interval'),
```

Comment:
We're switching to sentence case for column headers.

```suggestion
        title: t('catalog-source~Registry poll interval'),
```